src/native/corehost/hostpolicy/deps_resolver.cpp

```diff
@@ -43,29 +43,25 @@ void add_unique_path(
     pal::string_t* non_serviced,
     const pal::string_t& svc_dir)
 {
-    // Resolve sym links.
-    pal::string_t real = path;
-    pal::realpath(&real);
-
-    if (existing->count(real))
+    if (existing->count(path))
```

Comment:
We should probably add a comment admitting that in presence of symlinks or non-normalized paths there might be effective duplicates in the output - and that it's OK.